report.go

```diff
@@ -43,9 +43,17 @@ func report(results []*RunResult) {
 		if i == 0 {
 			reportFile.Title = folderPath
 		}
+
 		reportFile.Data[i].Name = name
 		reportFile.Data[i].HDR = read(filepath.Join(folderPath, name+".hdr"))
-		reportFile.Data[i].JSON = mustJSONUnmarshal(filepath.Join(folderPath, name+".json"))
+		json := mustJSONUnmarshal(filepath.Join(folderPath, name+".json"))
```

Comment:
I'm going to rename this. The function name is misleading, it does involve parsing JSON but it returns a `TestResult` value, so the variable name `json` is a bit confusing.